scripts/run-confluence-container.sh

```diff
@@ -32,18 +32,24 @@ esac
 # Set current folder to parent
 cd "$(dirname "$0")"/..
 
-if [[ ! -z "${CONFLUENCE_VERSION}" ]]
+#load default env varibles
+set -o allexport
+[[ -f .env ]] && source .env
+set +o allexport
+
+if [[ ! -z "${CONFLUENCE_RUN_VERSION}" ]]
 then
-    export CONFLUENCE_VERSION=${CONFLUENCE_VERSION}
+    export "CONFLUENCE_VERSION=${CONFLUENCE_RUN_VERSION}"
 fi
 
 for env_variable in ${args}
 do
- export ${env_variable}
- echo "set environment variable -> ${env_variable}"
+    export ${env_variable}
+    echo "set environment variable -> ${env_variable}"
 done
 
 echo "Starting Confluence version $CONFLUENCE_VERSION"
 echo "---------------------------------"
 
-docker-compose up confluence
\ No newline at end of file
+docker-compose up -d ${DATABASE} confluence
```

Comment:
So, what has basically changed is that the database is not a dependecy anymore , so we need to call explicitely.
On the other hand , we put -d as, now, it will show not only the confluence logs but also database ones.